src/main/java/roomescape/console/view/ConsoleInputView.java

```diff
@@ -0,0 +1,124 @@
+package roomescape.console.view;
+
+import java.time.DateTimeException;
+import java.time.LocalDate;
+import java.time.LocalTime;
+import java.time.format.DateTimeFormatter;
+import java.util.List;
+import java.util.Map;
+import java.util.Scanner;
+import roomescape.console.view.dto.DomainCommand;
+import roomescape.console.view.dto.FunctionCommand;
+import roomescape.domain.dto.ReservationRequest;
+import roomescape.domain.dto.ReservationTimeRequest;
+
+public class ConsoleInputView {
+
+    private static final Scanner SCANNER = new Scanner(System.in);
+
+    private static final String DELIMITER = ",";
+    private static final DateTimeFormatter DATE_FORMATTER = DateTimeFormatter.ofPattern("yyyy-MM-dd");
+    private static final DateTimeFormatter TIME_FORMATTER = DateTimeFormatter.ofPattern("HH:mm");
+
+    private static final Map<Integer, DomainCommand> INT_TO_DOMAIN_COMMAND = Map.of(
+            1, DomainCommand.RESERVATION, 2, DomainCommand.RESERVATION_TIME, 3, DomainCommand.END_PROGRAM);
```

Comment:
```suggestion
    private static final Map<Integer, DomainCommand> INT_TO_DOMAIN_COMMAND = Map.of(
        1, DomainCommand.RESERVATION,
        2, DomainCommand.RESERVATION_TIME,
        3, DomainCommand.END_PROGRAM);
```
intelliJ 가 어느정도 표시를 해주기는 하지만, 그래도 줄바꿈을 통해 map 의 key-value 묶음을 구분해봐도 좋을 것 같아요.
GitHub 에서 코드리뷰를 하는 사람에게도 이게 더 보기 편할테구요!